code/modules/clothing/spacesuits/alien.dm

```diff
@@ -388,7 +388,25 @@ obj/item/clothing/head/helmet/space/vox/civ/trader/stealth //blackhelmet
 	name = "vox roboticist pressure helmet"
 	icon_state = "vox-civ-roboticist"
 	desc = "A very alien-looking helmet for vox crewmembers. This one is for roboticists."
+	actions_types = list(/datum/action/item_action/toggle_helmet_mask)
+	var/up = 1
+	eyeprot = 0
 
+/obj/item/clothing/head/helmet/space/vox/civ/science/roboticist/attack_self()
```

Comment:
`attack_self` takes a `var/mob/user` as its first parameter, declare it and use it in the proc instead of `usr`